portals/publisher/source/src/app/components/Apis/Details/Policies/OperationPolicy.tsx

```diff
@@ -208,7 +210,7 @@ const OperationPolicy: FC<OPProps> = ({
 
     }, [apiOperations])
 
-    const handleExpansion = (panel: any) => (event:any, isExpanded:any) => {
+    const handleExpansion = (panel: any) => (event: any, isExpanded: any) => {
```

Comment:
Please avoid using `any` use a proper type definitions, `isExpanded` is a boolean ?